### ISSUE TITLE
error Still downloading .NET version(s) 8.0.6~x64

### ISSUE THREAD

**Issue by CodeBerserkers888** [dotnet/vscode-dotnet-runtime#1846]
### Describe the bug.

Despite an active internet connection, I get the ### error Still downloading .NET version(s) 8.0.6~x64

 ...Error when trying to start and install.
Failed to download .NET 8.0:
### Please attach log file(s) here if you have any.

2024-06-21 14:04:15.196 [error] Could not parse status output: "Running tunnel CLI\nstatus Spawning: c:\\Users\\AppData\\Local\\Programs\\Microsoft VS Code\\bin\\code-tunnel.exe tunnel status\nstatus exit(9268): + 0
2024-06-21 14:04:02.845 [info] update#setState idle
2024-06-21 14:04:32.832 [info] update#setState checking for updates
2024-06-21 14:04:32.868 [info] update#setState idle
2024-06-21 14:28:14.279 [info] Extension host with pid 4984 exited with code: 0, signal: unknown.
2024-06-21 14:28:40.399 [warning] [perf] profiling STARTED f91ca9d5-de0e-4361-b5ae-376ebc9a1b54
2024-06-21 14:28:45.453 [warning] [perf] profiling DONE f91ca9d5-de0e-4361-b5ae-376ebc9a1b54
2024-06-21 14:38:26.502 [info] Extension host with pid 10556 exited with code: 0, signal: unknown.

## 1.Check permissions
Make sure you have the proper permissions to install software on your computer. On Linux systems, you may need to use sudo:

bash
Copy the code
sudo apt install dotnet-sdk-8.0
5 Check the error logs
Sometimes detailed error logs can provide more information about the problem. Check the console for additional messages that may indicate what went wrong.

## Use a different installation tool
You can try using a different installation tool, such as snap on Ubuntu:

bash
Copy the code
sudo snap install dotnet-sdk --classic
7 Install the previous version
If the latest version causes problems, try installing the previous stable version of .NET.

### Expected Behavior

After following these steps, it should return to normal

### Exceptions & Errors (if any)

error Still downloading .NET version(s) 8.0.6~x64

### .NET Install Tool Version

v2.0.6

### Your operating system? Please provide more detail, such as the version and or distro above.

Windows

### VS Code Version

1.90.2

**Comment by nagilson** [comment]
Do you have a proxy? If so, we have an extension setting where you may need to set the proxy url.

**Comment by Alex-ley-scrub** [comment]
are these related? https://github.com/dotnet/vscode-dotnet-runtime/issues/1850

I see this message although `curl www.google.com` on the machine gives a response and I am connecting to the machine over ssh so it definitely has an internet connection.

<img width="1130" alt="image" src="https://github.com/user-attachments/assets/071795b3-98bd-4bd0-b446-58841a5db5ed">

**Comment by nagilson** [comment]
Yes, they are related. That offline check only runs after it's already failed so its just obfuscating the real reason it failed. If you try to create a new issue or open the extensions log folder and then go to our extension (runtime extension) log, the log will have the true reason for the failure

**Comment by Alex-ley-scrub** [comment]
I solved it by just running the command manually with sudo:
```sh
sudo "/home/alex/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-2.1.5/dist/install scripts/dotnet-install.sh" -InstallDir "/home/alex/.vscode-server/data/User/globalStorage/ms-dotnettools.vscode-dotnet-runtime/.dotnet/8.0.8~x64" -Version 8.0.8 -Verbose -Runtime dotnet -Architecture x64
```
next start up no longer shows the error message:
<img width="945" alt="image" src="https://github.com/user-attachments/assets/fec29d63-d7fb-49fc-8d1e-e8a398b9412a">
